src/requirements/data/colleges/en.ts

```diff
@@ -139,14 +140,15 @@ const engineeringRequirements: readonly CollegeOrMajorRequirement[] = [
   {
     name: 'Advisor-Approved Electives',
     description:
-      'At least 3 credit hours total. All academic courses count.' +
+      'At least 3 credit hours total. All academic courses count. ' +
       'No PE courses, courses numbered 10xx, and ROTC courses below the 3000-level allowed.',
     source:
       'https://www.engineering.cornell.edu/students/undergraduate-students/curriculum/undergraduate-requirement',
     checker: [
       (course: Course): boolean => {
-        const { catalogNbr } = course;
-        return !(ifCodeMatch(course.subject, 'PE') && ifCodeMatch(catalogNbr, '10**'));
+        if (courseIsSpecial(course)) return false;
```

Comment:
Why would the checker return `false` for a special course (i.e. an FWS, course that applies to total academic credits requirement) if all academic courses count?

Reply:
I don't think FWS courses count, and "All academic courses count" is a bit misleading.
![image](https://user-images.githubusercontent.com/34158284/113438121-f133ce80-93b5-11eb-86a5-29f74d04072e.png)

Reply:
I see. @einc is there a way to reform this sentence so that it's clearer for the users? Like "All academic courses, except FWS, count."

Reply:
If we reform the wording, we should also make sure to update the checkers. Currently the checkers allow regular FWS courses (other than our special FWS equivalent course) to fulfill the reqs because of the original wording.